src/js/components/Drop/DropContainer.js

```diff
@@ -155,7 +155,12 @@ class DropContainer extends Component {
           top = targetRect.bottom;
         }
 
-        if (windowHeight === top || windowHeight === targetRect.bottom) {
+        // Calculate visible area underneath the control w.r.t window height
```

Comment:
I'm trying to understand how the comment and the code go together. If the targetRect.bottom is 50 and the window height is 200, then the visibleAreaThreshold is 75. I'm trying to understand how 75 is either "the visible area underneath the control" or a "threshold". In the code further below 20 seems like a "threshold". Can you clarify a bit?

Reply:
If visibleAreaThreshold is 75% then it doesn't qualify for flipping the control. Consider that targetRect.bottom is 400  and window height is 480 then visibleAreaThreshold is close to 17% which means that it is within threshold limit that is defined here and there is not enough room to show the drop container, hence flip the control.

Reply:
Ah, so the 17% is a measure of the percentage of the available window height below the drop target. Can we call it something like `percentVisibleAreaBelow`?

Also, we should add a comment indicating why `20` was chosen as the threshold percentage.

Reply:
Updated as per your comments. Thanks for your review!!